### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Updated development section and minor editing for terminology and formatting.

### CODE REVIEW

**Review by larkox — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -5,32 +5,19 @@
...
 [![HW](https://img.shields.io/github/issues/mattermost/mattermost-plugin-zoom/Up%20For%20Grabs?color=dark%20green&label=Help%20Wanted)](https://github.com/mattermost/mattermost-plugin-zoom/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22Up+For+Grabs%22+label%3A%22Help+Wanted%22)
 
-
 **Maintainer:** [@larkox](https://github.com/larkox)
 **Co-Maintainer:** [@mickmister](https://github.com/mickmister)
 
-Start and join voice calls, video calls and use screen sharing with your team members via Zoom.
+Start and join voice calls, video calls and use screen sharing with your team members via Zoom. Check our [GitBook](https://mattermost.gitbook.io/plugin-zoom/) for further documentation about this plugin.
+
+## Usage and Set Up Guide
```
Comment (hanzei) [inline README.md:13]: Does this title really apply any longer?

File: README.md [hunk 2]
```diff
@@ -5,32 +5,19 @@
...
-## Usage & Setup Guide
 ### Commands
-* **`/zoom start`** &mdash; Start a zoom meeting.
-* **`/zoom disconnect`** &mdash; Disconnect from zoom (if OAuth is enabled).
-* **`/zoom help`** &mdash; Display help message.
 
-Check our [GitBook](https://mattermost.gitbook.io/plugin-zoom/) for further documentation about this plugin.
+* **`/zoom start`** &mdash; Start a Zoom meeting.
+* **`/zoom disconnect`** &mdash; Disconnect from Zoom (if OAuth is enabled).
+* **`/zoom help`** &mdash; Display help message.
```
Comment (hanzei) [inline README.md:19]: What is the value of having these here vs. just pointing everyone to the gitbook docs?

**Review by hanzei — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (justinegeffen) [reply README.md:19]: Good question, and no value. I'm reviewing this now.

File: README.md (hunk 1 above)
Comment (justinegeffen) [reply README.md:13]: Nope, good point.

**Review by hanzei — APPROVED** [review]
LGTM :+1:

**Pull request merged**